README.md

```diff
@@ -120,3 +154,14 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 <!-- ALL-CONTRIBUTORS-LIST:END -->
 
 This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
+
+
+## Acerca de EOS Costa Rica
+
+<div style={{ display: "block", textAlign: "center" }}>
+    <img style={{ width: "50%" }} src="https://raw.githubusercontent.com/eoscostarica/design-assets/master/logos/eosCR/fullColor-horizontal-transparent-white.png" />
```

Comment:
<img width="50%"

Reply:
usemos atributos html para el ancho de la imagen (creo que github no entiende `style={{ width: "50%" }}` )

Reply:
> 
> 
> usemos atributos html para el ancho de la imagen (creo que github no entiende `style={{ width: "50%" }}` )

You right, i just test it, and yes, it don't understand that way